drivers/adc/adc_mcux_lpadc.c

```diff
@@ -122,10 +122,16 @@ static int mcux_lpadc_acquisition_time_setup(const struct device *dev, uint16_t
 static int mcux_lpadc_channel_setup(const struct device *dev,
 				const struct adc_channel_cfg *channel_cfg)
 {
+	const struct mcux_lpadc_config *config = dev->config;
+	const struct device **regulator = config->ref_supplies;
+	uint16_t vref_mv = CONTAINER_OF(channel_cfg, struct adc_dt_spec, channel_cfg)->vref_mv;
+	int32_t min_uv = (int32_t)((uint32_t)vref_mv * 1000);
+	int32_t max_uv = (int32_t)((uint32_t)vref_mv * 1000);
```

Comment:
why have two variables if they are the same?

Reply:
Considering saving stack, using one variable in the current situation is ok, I will update it.